code/src/components/Form.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+const Form = ( { onFormSubmit, newThought, onSetThoughtChange } ) => {
+
+    return (
+       <section className='form-container'>
+            <form 
+                onSubmit={onFormSubmit}
+                className='form-content'
+            >
+                <h1>What's making you happy right now?</h1>
+                <label htmlFor='new-thought'>
+                    <textarea 
+                        className='input-text'
+                        rows="5"
+                        cols="30"
+                        onChange={onSetThoughtChange}
+                        value={newThought}
+                        id='new-thought'
+                        name='new-thought'
+                        placeholder='A penny for your thought!'
```

Comment:
Nice detail with the htmlFor, and clever approach with the rows and cols here instead of the CSS!
I really like your placeholder text also, makes me want to type something :)